PORTAL-3078: Shared Catalogs end-user documentation (#43)
akeneo/portal-helpcenter: content/md/shared-catalogs/1-set-up-shared-catalogs.md

---
id: set-up-shared-catalogs
themes: manage-shared-catalogs-instance
title: "**Set up** your Shared Catalogs instance"
popular: true
related: console-activity-history
---

# Set up your Shared Catalogs instance

The Akeneo Shared Catalogs instance creation is managed from the **project console** of the [Akeneo Portal](https://portal.akeneo.com). 

Following the purchase of a Shared Catalogs instance, an email is sent to the **Shared Catalogs administrator** inviting them to fill in the setup form to create the instance.

You can set up the instance from the link in the email or by accessing the instance configuration page on the **project console**.

![Shared Catalogs list](../img/shared_catalogs_list.png)

1. First, from the menu, go to `Project console / Projects` and select your project.
2. Go to the Shared Catalogs instance tab, and open the instance with the status `Waiting for setup`, by clicking on the instance line.
3. Then fill in the form with the missing prerequisites in order to create the Shared Catalogs instance.

## Link your Shared Catalogs to an existing PIM Cloud instance

> 1. Select a `Linked PIM instance` in the list.

Reviewer: You may add that it will pre-fill the installation continent field?

2. Choose an `Instance name`, it will not be a domain name.
3. Input your `PIM API connection credentials`, test the connection to validate.
4. Once the form is filled in and connection successfully tested, click on `Request an instance`.

![Shared Catalogs setup form](../img/shared_catalogs_setup.gif)

> ::: warning
> If the linked PIM is a Flexibility instance it should be compatible only with 4.0 and above version.
> :::

Reviewer 2: ```suggestion
::: warning
If the linked PIM is a Flexibility instance, the Shared catalogs are compatible only with PIM version 4.0 and above.
:::
```
